.github/workflows/pr-label.yml

```diff
@@ -34,7 +36,7 @@ jobs:
       pull-requests: write
     runs-on: ubuntu-latest
     steps:
-      - uses: codelytv/pr-size-labeler@v1.7.0
+      - uses: codelytv/pr-size-labeler@v1
         with:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           s_label: "size/s"
```

Comment:
Add an XS with the one line label

Reply:
I have added an XS label for ≤2 lines (allows whitespace)